src/server/pfs/server/driver.go

```diff
@@ -476,17 +513,43 @@ func (d *driver) FinishCommit(ctx context.Context, commit *pfs.Commit) error {
 			if err := tree.DeleteFile(filePath); err != nil {
 				// Deleting a non-existent file in an open commit should
 				// be a no-op
-				if !(hashtree.Code(err) == hashtree.PathNotFound) {
+				if hashtree.Code(err) != hashtree.PathNotFound {
```

Comment:
Freed up a pair of parentheses and an equals sign for use elsewhere in the cluster.